server/README.md

```diff
@@ -206,6 +211,32 @@ Further information can be found at: http://ruby-for-beginners.rubymonstas.org/b
 
 ## Deployment
 
+Using Heroku to host production instance. 
+
+Once logged into Heroku code can be deployed by running: 
+
+```
+git push heroku whatever_branch_I_am_on_branch:master
+```
+Note that you cannot push to any branch other than master on Heroku, although the branch on the local machine can be anything. 
+
+Because our project folder contains two sub projects in different languages, deployment requires setting the folder to be deployed to production in the buildpack. 
+
+Instructions for getting around this issue were taken from this tutorial: https://elements.heroku.com/buildpacks/techgaun/heroku-buildpack-subdir-to-root
+
+The key commands to be executed in route directory of sub-project (in my case /server) are: 
+
+```
+heroku buildpacks:add --index 1 https://github.com/techgaun/heroku-buildpack-subdir-to-root.git --app mentor-mentee-platform
+heroku config:set PROJECT_RELATIVE_PATH=packages/adminWeb --app mentor-mentee-pkatform
+```
+
+After you deploy the app, you need to seed the database in order for swagger docs to work. 
```

Comment:
This section made me think of https://devcenter.heroku.com/articles/release-phase - a Heroku feature.

As a future improvement, perhaps we can use that, for tasks to always run just before being done with deployment.

Reply:
Added Issue #259  to automate running tasks when deploying in Heroku.